internal/services/authorizer/testutils.go

```diff
@@ -0,0 +1,28 @@
+package authorizer
+
+// All those functions and methods are only for tests.
+// They are not exported, and guarded by testing assertions.
+
+import (
+	"fmt"
+	"os/user"
+	"strconv"
+
+	"github.com/ubuntu/authd/internal/testsdetection"
+)
+
+// currentUserUID returns the current user UID or panics.
+func currentUserUID() uint32 {
```

Comment:
Couldn't be this part of `testutils` instead and be a `t.Helper()` to handle errors?

Reply:
As long as we don’t have consumers outside of this package, I’m a little bit reluctant to add it to `testutils`. If we have multiple direct consumers, yeah, we can migrate it over.